CmdStarter.Tests/Options.cs

```diff
@@ -126,5 +128,42 @@ public void MethodSetup()
                 Assert.That(option, Is.Null);
             }
         }
+
+        [TestCase<OptByAttribute>]
+        [TestCase<OptByIncludeAttribute>]
+        [TestCase<OptByExcludeAttribute>]
+        public void EnsureOptionAttributes<OptClass>() where OptClass : class, IStarterCommand, IOptByAttribute
+        {
+            starter.Namespaces = starter.Namespaces.Add(typeof(OptClass).Namespace!);
+            starter.InstantiateCommands();
+
+            var optionCommand = starter.FindCommand<OptClass>();
+            Assert.That(optionCommand, Is.Not.Null);
+
+            foreach (var includedOption in OptClass.IncludedOptions)
+            {
+                Assert.That(optionCommand.Options.All(option =>
+                    option.Name.Equals(includedOption.PascalToKebabCase())), Is.True);
+            }
+
+            foreach (var excludedOption in OptClass.ExcludedOptions)
+            {
+                Assert.That(optionCommand.Options.Any(option =>
+                    option.Name.Equals(excludedOption.PascalToKebabCase())), Is.False);
+            }
+        }
+
+        [TestCase<OptAllPropertiesExcluded>(OptAllPropertiesExcluded.OPTION_COUNT)]
+        [TestCase<OptAllPropertiesExcludedWithInheritance>(OptAllPropertiesExcludedWithInheritance.OPTION_COUNT)]
+        public void EnsureClassExcludeAttribute<OptClass>(int optionCount) where OptClass : class, IStarterCommand
```

Comment:
This is good, but a simple question. Why not using the new test method above?

Reply:
What do you mean exactly?

Why I didn't do this in that method or why I didn't copy a logic from there?

Reply:
I mean: why those test cases aren't tested using `EnsureOptionAttributes` ?

Reply:
Oh, I see!

Well... good question hehe

Reply:
Done